debian/control

```diff
@@ -0,0 +1,25 @@
+Source: pymavlink
+Section: python
+Priority: optional
+Maintainer: Andrii Beregovenko <jet@jet.kiev.ua>
+Build-Depends: debhelper (>= 10),
+  dh-python,
+  python3-all-dev (>= 3.3.3),
+  libxml2-dev, libxslt1-dev, zlib1g-dev,
+  python3-setuptools (>= 0.6.29),
+  python3-numpy,
+  python3-pytest,
```

Comment:
I'm not sure we have a hard dep for this.

Reply:
ok, fixed